### DISCUSSION TITLE
Will I lose access to GitHub Pro if I change my primary email from my academic email to a non-academic email?"

### DISCUSSION THREAD

**Discussion by omridinio** [community/community#133916] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I currently have access to GitHub Pro through my academic email. I'm planning to change my primary email to a non-academic email as I will be losing access to my academic email soon. 

Will I lose access to GitHub Pro if I change my primary email to a non-academic email? If so, is there any way to maintain my GitHub Pro benefits after changing my primary email?

Thank you for your assistance.

**Comment by harisdev-netizen** [comment, marked as answer]
If you change your primary email from an academic to a non-academic email, you will likely lose access to GitHub Pro, as the benefits are tied to your academic status and verified through your academic email.

However, to maintain your GitHub Pro benefits, you have a couple of options:

1. Verify New Academic Email: If you have access to another academic email, you can update your GitHub account with the new academic email and re-verify your student status to continue enjoying the GitHub Pro benefits.
2. GitHub Student Developer Pack: If you are still a student, you can reapply for the GitHub Student Developer Pack using your new academic email. The pack includes GitHub Pro and other useful tools for students.
3. GitHub Education Support: You can contact GitHub Education Support and explain your situation. They might offer solutions or extend your benefits temporarily until you obtain a new academic email.

**Reply by omridinio** [reply]
Hello,

Thank you for the information. I have a follow-up question:

What happens to my GitHub account access after I graduate and my academic email is deactivated? Will my primary email change automatically to the secondary email I added, or do I need to manually change my primary email before my academic email is deactivated?

Thank you for your assistance.

**Reply by harisdev-netizen** [reply]
After you graduate and your academic email is deactivated, your primary email on GitHub will not automatically change to the secondary email you have added. You will need to manually update your primary email before your academic email is deactivated to ensure continuous access to your GitHub account.

**To change your primary email on GitHub:**
1. Go to your GitHub account settings.
2. Navigate to the “Emails” section.
3. Add your new email address if you haven't already.
4. Verify the new email address.
5. Set the newly verified email as your primary email.

**Reply by harisdev-netizen** [reply]
Please mark my reply as **Mark as Answer** if you found it helpful. Thank you!!
